translations/readme/README.jp.md

```diff
@@ -1,5 +1,12 @@
 # 🟡 GROC &middot; ![GitHubリポジトリサイズ][1] ![GitHubのコードサイズ（バイト）][2]
 
+## 他の言語で読む [こちら](translations/readme/).
+
+![🇦🇩 Catalan][3] ·
```

Comment:
Translate the languages list into its own language, but keep this order (as if they were showed in English).